File: KELAS11/CODING/APCSA/0829/Q03.py
class Tree:

    def __init__(self, n=None, h=None, r=None):
        self.name = n
        self.height = h
        self.rate_of_growth = r

    def __repr__(self):
        return f"This tree has name, and it's {self.name} ..."
        

tree_1 = Tree("akasia", 200, 0.2)
tree_2 = Tree(300, 0.3)
tree_3 = Tree("mahoni")
print(tree_1)
print(tree_2)
print(tree_3)

File: KELAS11/CODING/APCSA/0829/Q03.java
public class Q03{
    
    public static void main(String[] args)
    {
        Tree treeOne = new Tree("akasia", 200, 20);
        System.out.println(treeOne.toString());

        Tree treeTwo = new Tree(300, 30);
        System.out.println(treeTwo);

        Tree treeThree = new Tree("mahoni");
        System.out.println(treeThree);

        Tree treeFour = new Tree();

        // Tree treeFour = new Tree();
        // System.out.println(treeFour);
    }
}